roles/ceph-common/tasks/misc/system_tuning.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: disable osd directory parsing by updatedb
+  command: updatedb -e /var/lib/ceph
+  changed_when: false
+  ignore_errors: true
+
+- name: disable transparent hugepage
+  command: "echo never > /sys/kernel/mm/transparent_hugepage/enabled"
```

Comment:
This only modifies runtime configuration, if this is required we should ensure it is applied at boot too.